### PR TITLE
Add Modal css

### PR DESCRIPTION
**Pull request by rahulcs754**
Added model CSS

image

Please review

### CODE REVIEW

File: css/components/modal.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/*
+|| Modal 
+*/
+.modal {
+    position: fixed;
+    z-index: 1;
```
Comment (tanishq20) [inline css/components/modal.css]: Why these both properties are needed.

File: css/components/modal.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+.modal-l{
+    width: 60%;
+    max-width: 80%;
+}
+
+/* Model Close Button */
+.modal-close {
+    position: absolute;
+    top:0px;
+    right: 1rem;
```
Comment (shuntzhere) [inline css/components/modal.css]: Try to use either px or rem for positioning. This may create some problems.

File: css/components/modal.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    position: relative;
+    background-color: var(--mint-white);
+    margin:0 auto;
+    padding: 0;
+    border: 1px solid var(--light-more-black);
+    box-shadow: 0px 0px var(--light-black);
+    border-radius: 10px;
+}
+
+
```
Comment (shuntzhere) [inline css/components/modal.css]: An extra line space can be removed.

**Review by shuntzhere — CHANGES_REQUESTED** [review]
Suggested a change that may create problems with alignment later.

**Review by tanishq20 — APPROVED** [review]

**Review by shuntzhere — APPROVED** [review]
It looks good now.

**Comment by rahulcs754** [comment]
image
I need a black part to cover the whole viewport that why  used display :fixed

second i need my place wall alwage on top to element
I think now you can understand

**Pull request merged**